corehq/apps/data_cleaning/templates/data_cleaning/clean_cases_session.html

```diff
@@ -2,17 +2,24 @@
 {% load hq_shared_tags %}
 {% load django_tables2 %}
 {% load i18n %}
-
 {% js_entry "data_cleaning/js/clean_cases_session" %}
 
 {% block content %}
   {% breadcrumbs current_page section current_page.parents %}
   <div class="d-flex align-items-center m-3">
-    <h1 class="fs-3 pe-3 py-3 m-0">
-      {{ current_page.page_name }}
-    </h1>
+    <h1 class="fs-3 pe-3 py-3 m-0">{{ current_page.page_name }}</h1>
```

Comment:
disagree with the formatting change. i think it's more readable to have `{{ current_page.page_name }}` on its own line. can you please revert?

Reply:
(seems like this shouldn't have been included with the same commit to add the temporary save button?)

Reply:
I just ran prettify on the file. I don't think it's sustainable to have coding standards that our tooling overwrites.